src/styles/blocks/card.scss

```diff
@@ -0,0 +1,62 @@
+.card {
+  box-sizing: border-box;
+  position: relative;
+  text-align: justify;
+  width: 200px;
+  height: 407px;
+  border: 1px solid $grey-star-color;
+  border-radius: 5px;
+  transition: all 0.3s linear;
+}
+
+.card__product {
+  display: block;
+  padding-inline: $standard-inline-pad;
+  padding-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: $standard-black-color;
```

Comment:
Ensure that the variable `$standard-black-color` is correctly spelled and consistent with the naming conventions.